hw/top_darjeeling/defs.bzl

```diff
@@ -16,4 +16,8 @@ DARJEELING = opentitan_top(
     std_otp_overlay = DARJEELING_STD_OTP_OVERLAYS,
     otp_sigverify_fake_keys = DARJEELING_OTP_SIGVERIFY_FAKE_KEYS,
     ips = DARJEELING_IPS,
+    secret_cfgs = {
```

Comment:
Tangential comment to this issue, but I missed reviewing the previous PRs...
Can I bikeshed on the names of the seed groups / modes? 'dev' and 'prod' could be quite confusing given they exactly match lifecycle states. Could I suggest 'testing' and 'tapeout' to begin with? Not sure if there is an appropriate issue to bundle this into, but I could create one..

Reply:
Fixed in #28089